src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -5933,8 +5933,8 @@ static void setDrumset(Chord* c, MusicXMLParserPass1& pass1, const String& partI
     pass1.setDrumsetDefault(partId, instrumentId, headGroup, line, overruledStemDir);
 }
 
-static void xmlSetDrumsetPitch(Note* note, const Chord* const chord, const Staff* const staff, int step, int octave,
-                               NoteHeadGroup headGroup, const Instrument* const instrument)
+static void xmlSetDrumsetPitch(Note* note, Chord* const chord, const Staff* const staff, int step, int octave,
+                               NoteHeadGroup headGroup, DirectionV& stemDir, const Instrument* const instrument)
```

Comment:
Is it really necessary to have all these const-pointer to const-object definitions? Can't it just be `const Chord*` `const Staff*` `const Instrument*`?